### ISSUE TITLE
Dismount button not working

### ISSUE THREAD

**Issue by anaarkei** [GoldpawsStuff/AzeriteUI#34]
Clicking on the dismount button on the minimap does nothing. Other, less convenient, methods of dismounting must be utilised.

**Comment by deftanksfault** [comment]
I have the same issue.

**Comment by anaarkei** [comment]
I disabled all other add-ons to see if the problem was caused by conflict however there was no change in behaviour, the button still does nothing.

**Comment by P154** [comment]
> 

Yes. I've checked the addon standalone and coupled with my currently used addons. The dismount button appears to be working in order for me.

**Comment by deftanksfault** [comment]
Hm that's odd! Are you playing classic or retail?

**Comment by P154** [comment]
Retail

**Comment by deftanksfault** [comment]
Ok 🤷‍♂️

**Comment by qq5570461** [comment]
> I disabled all other add-ons to see if the problem was caused by conflict however there was no change in behaviour, the button still does nothing.

I have the same issue.
